### PR TITLE
Deployment script :rainbow: 

### PR DESCRIPTION
**Pull request by yvan-sraka**

### CODE REVIEW

**Comment by AureliaDolo** [comment]
For now this is just installing python somewhere ?

**Comment by yvan-sraka** [comment]
> For now this is just installing python somewhere ?

And deploy script dependencies using Poetry lock file :)

**Review by AureliaDolo — APPROVED** [review]

**Comment by yvan-sraka** [comment]
bors merge

**Comment by yvan-sraka** [comment]
bors merge

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -61,4 +61,16 @@ jobs:
...
+
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: "recursive"
+    - run: |
+       apt install python3 poetry
+       cd massa-deploy
```
Comment (qdrn) [inline .github/workflows/cd.yml:74]: The deploy script for the testnet is no longer in the massa-deploy directory.
```suggestion
       cd massa-deploy/testnet
```

**Review by qdrn — COMMENTED** [review]
Except the directory it looks good to me!

**Review by qdrn — APPROVED** [review]
Looks good to me. Did you add the relevant secrets ?

**Comment by yvan-sraka** [comment]
> Looks good to me. Did you add the relevant secrets ?

I don't have the secrets @damip they should be add here https://github.com/massalabs/massa/settings/secrets/actions

**Review by qdrn — APPROVED** [review]

**Comment by yvan-sraka** [comment]
bors merge

**Pull request merged**